OrganizationSearch/OrganizationSearch.js

```diff
@@ -55,6 +55,7 @@ class OrganizationSearch extends Component {
               onClick={this.openModal}
               ariaLabel={ariaLabel}
               tabIndex="-1"
+              {...buttonProps}
```

Comment:
@aditya-matukumalli what props do you want to pass? I don't really like object props

Reply:
It's not actually about `to like something`, but it's unclear and unpredictable what could go in this object. As a last resort it could be described in `propTypes` as `PropTypes.shape()`, IMHO

Reply:
@NikitaSedyx @aliaksei-chumakou I would like to pass the _marginBottom0_ prop down to the Button

Reply:
I'm OK with your way, thanks

Reply:
@NikitaSedyx @aliaksei-chumakou like Aditya said, for now it's just `marginBottom0`, but it's also nice to be able to pass `data-test` attributes buttons like this.

Ultimately, it'd be best to be able to fully customise the rendering of this the way that a few plugins already allow, such as [`ui-plugin-find-license` does with the `renderTrigger` prop](https://github.com/folio-org/ui-plugin-find-license/blob/master/src/LicenseSearch.js)

Reply:
@doytch render props works for me, it's much batter than set of props to customize button

Reply:
but it can be breaking change